target/linux/mediatek/image/mt7622.mk

```diff
@@ -239,3 +239,19 @@ define Device/ubnt_unifi-6-lr-ubootmod
   ARTIFACT/bl31-uboot.fip := bl31-uboot ubnt_unifi-6-lr
 endef
 TARGET_DEVICES += ubnt_unifi-6-lr-ubootmod
+
+define Device/xiaomi_redmi-router-ax6s
+	DEVICE_VENDOR := Xiaomi
+	DEVICE_MODEL := Redmi Router AX6S
+	DEVICE_ALT0_VENDOR := Xiaomi
+  DEVICE_ALT0_MODEL := Router AX3200
+	DEVICE_DTS := mt7622-xiaomi-redmi-router-ax6s
+	DEVICE_DTS_DIR := ../dts
+	DEVICE_PACKAGES := kmod-mt7915e
+	KERNEL := kernel-bin | lzma
+	KERNEL_INITRAMFS_SUFFIX := -recovery.itb
+	KERNEL_INITRAMFS := kernel-bin | lzma | fit lzma $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb with-initrd | pad-to 128k
```

Comment:
Please wrap after 74 chars. max.

Reply:
Fixed, initramfs image didn't need padding either.

I note that there's a couple other contraventions of this in there, but that's out of scope for this patch/pr.